### ISSUE TITLE
Failed to find exported name of node

### ISSUE THREAD

**Issue by rcketscientist** [angular/angular#29441]
# 🐞 bug report

### Affected Package

 The issue is caused by package @angular/....

`ivy-ngcc` throws this:
`Failed to find exported name of node`
on an internal module that's not exported in a library.  While I'm sure this is some minor issue in the library that's gone undetected until now, I can't really gather what is wrong from the error message.  The transpiled code for the class in question doesn't differ from other classes.

Edit:
---
I removed the initial class entirely and got the same error on a traditional exported module:
```
Error: Failed to find exported name of node (function AlloyDialogModule() {
    }) in 'C:/Keysight/Development/alloy-window-manager-feasibility/node_modules/@keysight/alloy/esm5/lib/dialog/dialog.module.js'.
    at Object.findExportedNameOfNode (C:\Keysight\Development\alloy-window-manager-feasibility\node_modules\@angular\compiler-cli\src\ngtsc\imports\src\find_export.js:33:19)d_export.js:33:19)    
```
---

Would appreciate a pointer.  Thanks!

## 🌍  Your Environment

**Angular Version:**
<pre><code>

Angular CLI: 8.0.0-beta.7Node: 10.14.2
OS: win32 x64
Angular: 8.0.0-beta.9
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.800.0-beta.7
@angular-devkit/build-angular      0.800.0-beta.7
@angular-devkit/build-ng-packagr   0.800.0-beta.7
@angular-devkit/build-optimizer    0.800.0-beta.7
@angular-devkit/build-webpack      0.800.0-beta.7
@angular-devkit/core               8.0.0-beta.7
@angular-devkit/schematics         8.0.0-beta.7
@angular/cdk                       7.3.3
@angular/cli                       8.0.0-beta.7
@angular/material                  7.3.3
@ngtools/json-schema               1.1.0
@ngtools/webpack                   8.0.0-beta.7
@schematics/angular                8.0.0-beta.7
@schematics/update                 0.800.0-beta.7
ng-packagr                         4.7.1
rxjs                               6.4.0
typescript                         3.3.4000
webpack                            4.29.6
</code></pre>

**Comment by rcketscientist** [comment]
While "solving" another ivy compiler issue I messed up an import which broke the package metadata and led to this.  The library ran in consumers, but still had issues.  I don't believe this is a legitimate issue.

**Comment by christianrondeau** [comment]
For the record, I have the same issue but with the `ngx-charts` component. Since the actual error message only gives this error as a result, here's the related issue: https://github.com/angular/angular/issues/30512

The problem is that `PieGridSeriesComponent` is not exported in `index.ts`. I'll try and provide a PR.

**Comment by vdumbrav** [comment]
@christianrondeau 
**swimlane** has fixed the error in version 12.0.1 (ngx-chart)
